gr-qtgui/include/gnuradio/qtgui/waterfall_sink_c.h

```diff
@@ -98,7 +98,7 @@ class QTGUI_API waterfall_sink_c : virtual public sync_block
      *        the PDU message port is being used.
      * \param parent a QWidget parent object, if any
      */
-    static sptr make(int size,
+    static sptr make(int fftsize,
```

Comment:
Changing the parameter name to `fftsize` is correct/consistent with the purpose and implementation. The doxygen documentation on line 88 needs to be updated as well to match.

The GRC YAML already uses FFT Size.
https://github.com/gnuradio/gnuradio/blob/master/gr-qtgui/grc/qtgui_waterfall_sink_x.block.yml#L21